03-python2-errors-instructors.ipynb

```diff
@@ -0,0 +1,619 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Errors and Exceptions"
```

Comment:
Gonna put the 'Introduction to Python 2' heading in here for consistency with the other lesson.